sdks/python/apache_beam/tools/distribution_counter_microbenchmark.py

```diff
@@ -47,21 +46,22 @@ def run_benchmark(num_runs=100, num_input=10000, seed=time.time()):
   lower_bound = 0
   upper_bound = sys.maxint
   inputs = generate_input_values(num_input, lower_bound, upper_bound)
+  distribution_counter = __import__(
+      'apache_beam.transforms.distribution_counter', globals(), locals(), -1)
```

Comment:
<!--new_thread; commit:53a4caa11c9ea2ff223d462eb636791164b55621; resolved:0-->
Why do we need to do this strange form of import?

Reply:
distribution_counter can be imported only when Cython available. So import distribution_counter at the beginning may raise ImportError.